events/bus.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // EventBus manages the state of and transmits messages to all its Subscribers
 type EventBus struct {
-	registry map[Subscriber]bool
+	registry map[*Subscriber]bool
```

Comment:
`Subscriber` was an interface, not an instance. You can't take a pointer to an interface, so when we switched here to using a concrete object we want to make sure that's a reference so that it can unsubscribe itself later.